packages/extension-base/src/background/handlers/chromeStorage.ts

```diff
@@ -1,61 +1,32 @@
 import { z } from 'zod';
 
-import { PASSWORD_EXPIRY_MS } from '../../defaults';
+// use namespaces to add typization for your storage data
+const namespaces = { 'password-cache': z.record(z.string(), z.object({ password: z.string(), expiresAt: z.number() })) } as const satisfies Record<string, z.ZodSchema>;
```

Comment:
Dang, I should've checked if something like that existed already.

While reusing it for session storage is of course a good idea, I think it can blow up this PR. Let's maybe consolidate it in a follow-up PR?